solr/core/src/java/org/apache/solr/core/FileSystemConfigSetService.java

```diff
@@ -259,6 +260,14 @@ public FileVisitResult postVisitDirectory(Path dir, IOException ioException) {
     return filePaths;
   }
 
+  private String normalizePathToForwardSlash(String path) {
+    return path.replace(File.separatorChar, '/');
+  }
+
+  private String normalizePathToOSSeparator(String path) {
+    return path.replace('/', File.separatorChar);
```

Comment:
Instead of referencing the old `java.io.File`, please use `configSetBase.getFileSystem().getSeparator()`

Reply:
Fixed in 836f33ad38a2bcca839e37d63ff35a78acbee404